knowledge-base/failed-to-create-component-reportviewer-after-upgrade.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Failed to create component ReportViewer
+description: Report viewer crashes when trying to place into form after upgrade
+type: troubleshooting
+page_title: Failed to Create Component ReportViewer After Upgrade
+slug: failed-to-create-component-reportviewer-after-upgrade
+position: 
+tags: reportviewer, standalonedesigner,reportdesigner, upgrade
+ticketid: 1557422
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+**Failed to create component ReportViewer** error message might appear in the Visual Studio Report Designer after upgrade of Telerik Reporting version.
+
+## Error Message
+Failed to create component ReportViewer. The error message contains follows: "System.IO.FileNotFoundException:
```

Comment:
Change 'follows' to something meaningful in:
The error message contains follows: "System.IO.FileNotFoundException: